addons/product/views/product_attribute_views.xml

```diff
@@ -92,6 +92,20 @@
         </field>
     </record>
 
+    <record id="product_template_attribute_line_tree" model="ir.ui.view">
```

Comment:
```suggestion
    <record id="product_template_attribute_line_view_tree" model="ir.ui.view">
```
update name as well


Also create new file to add this view `product_template_attribute_line_view` :slightly_smiling_face: